src/SingleCellAnnotator.imjoy.html

```diff
@@ -805,6 +817,7 @@
       // annotation viewer
       this.current_panel = 'imageviewer';
       if(!this.draw_mask_mode){
+        this.loading = true;
```

Comment:
this.loading = true, those adding is to block the savamaskannotation before this.loading = false

Reply:
This has no effect though, since stopMaskAnnotation is not an async function.
This part is not safe, since you need to also consider that stopMaskAnnotation and the subsequent operation can fail, if that happens, you will end up with a loading animation running forever. (see my example at line 838, I set false in the `finally` block which make sure we will have it closed).